site/content/en/docs/Third Party Content/examples.md

```diff
@@ -25,4 +25,14 @@ You will need to download the Minetest client separately to play.
 * [Quilkin](https://github.com/googleforgames/quilkin) is a non-transparent UDP proxy specifically designed for use with large scale multiplayer dedicated game server deployments, to ensure security, access control, telemetry data, metrics and more.
 * [Quilkin with Agones](https://github.com/googleforgames/quilkin/tree/main/examples) is an example of running [Xonotic](https://xonotic.org/) with Quilkin on an Agones cluster, utilising either [the sidecar integration pattern](https://github.com/googleforgames/quilkin/tree/main/examples/agones-xonotic-sidecar) or via the the [Quilkin xDS Agones provider](https://github.com/googleforgames/quilkin/tree/main/examples/agones-xonotic-xds) with a TokenRouter to provide routing and access control to the allocated GameServer instance.
 
-You will need to download the Xonotic client to interact with the demo.
\ No newline at end of file
+You will need to download the Xonotic client to interact with the demo.
+
+## Shulker
+
+- [Shulker](https://github.com/jeremylvln/Shulker) is a Kubernetes operator for managing complex and dynamic Minecraft
+  infrastructure at scale, including game servers and proxies. 
+- It builds on top Agones `GameServer` and `Fleet` primitives to provide simplified abstractions specifically tailored
+  to orchestrating Minecraft workloads. 
+
+Shulker requires you to have a genuine Minecraft account. You'll need to [purchase the game](https://www.minecraft.net/en-us/article/how-create-minecraft-account)
```

Comment:
```suggestion
Shulker requires you to have a genuine Minecraft account. You'll need to <a href="https://www.minecraft.net/en-us/article/how-create-minecraft-account" data-proofer-ignore>purchase the game</a>
```

Looks like the website is actively blocking us from testing the link, so we'll ignore it.

Otherwise, this looks good to go once we can see a preview 👍🏻

Reply:
Done :) Let's see if the CI passes :)